server/eclipse-project/src/main/java/us/freeandfair/corla/controller/ContestCounter.java

```diff
@@ -152,13 +196,14 @@ public static Integer margin(final Map<String,Integer> voteTotals) {
     return rankTotals(voteTotals).stream()
       .limit(winnersAllowed)
       .map(Entry::getKey)
-      .map(k -> k.toString()) // types?
+      .map(k -> k.toString()) // Satisfy the typechecker: getKey returns an Object
```

Comment:
You could eliminate this coercion + comment if you adjust the return type signature of `rankTotals` to use `Entry<String, Integer>` rather than `Entry`.

Reply:
You're right! 👍